tests/test_integration_default.py

```diff
@@ -283,18 +331,18 @@ def test_unused_snapshots_ignored_if_not_targeted_when_targeting_using_dash_k(st
     testdir.makefile(
         ".py", **{"test_life_always_finds_a_way": test_content},
     )
+    testfile = os.path.join(testdir.tmpdir, "test_life_always_finds_a_way.py")
     result = testdir.runpytest(
-        "-k", "life_always_finds_a_way", "-v", "--snapshot-update"
```

Comment:
This was already covered by `test_collection` so repurposed to test for the node ids case